CIP-0107/README.md

```diff
@@ -0,0 +1,108 @@
+---
+CIP: 107
+Title: URI Scheme - Block and transaction objects
+Status: Proposed
+Category: Tools
+Authors:
+    - Pi Lanningham <pi@sundaeswap.finance>
+Implementors: N/A
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/556
+Created: 2020-09-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This extends CIP-13, which describes a Cardano URI scheme, with several more objects that would be useful to be able to assign addresses to, in particular blocks, transactions, transaction metadata, and transaction outputs.
+
+## Motivation: why is this CIP necessary?
+
+CIP-13 defined two initial URL authorities, for payment links and delegating to a stakepool.
+
+However, in a number of contexts, it would be useful to canonically link to other Cardano objects, such as:
+- Providing links to a transaction, to be opened in a wallet or chain explorer of the users choice
+- To provide richly interconnected metadata, such as in [CIP-100](https://github.com/cardano-foundation/CIPs/pull/556)
+
+Without a canonical standard for how to define these URIs, these objects are either unaddressable, not machine-interpretable, or will suffer from a divergence of convention. Similarly, we seek to fit an existing structure, such as CIP-0013, to reduce the number of different conventions that need be supported by the ecosystem.
+
+## Specification
+
+We extend CIP-13 with 2 new authorities for referencing blocks and transactions.
+
+Examples:
+```
+web+cardano://block/c6a8976125193dfae11551c5e80a217403d953c08ebbd69bba904d990854011f
```

Comment:
(1 of 2) If `height=` is preserved in the query part of the URI syntax documented below, an example should be added here.

Reply:
p.s. should be considered along with @Crypto2099's https://github.com/cardano-foundation/CIPs/pull/635#discussion_r1424224364.